list.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'my_enumerable'
```

Comment:
- [ ] I got an error loading the file name `my_enumerable` because it doesn't exist. I believe this is a typo because the correct file name is `enumerable`. Try to update the file name.

See screenshot
![a1](https://user-images.githubusercontent.com/39523312/155805104-5d6b52ce-78c0-4d9d-bff8-40a4e7c4de2f.png)

Reply:
done